content/04.experiment-types.md

```diff
@@ -16,4 +16,146 @@ Proteomics experiments can be both qualitative and quantitative.
+
+Following are some suggestive steps as to creating a XL-MS workflow:
+1. Select ion of interest if it is targeted proteomics.
+2. Cross-link the molecular ion through a cross-linking agent (DSS, BS3, EDC, DMTMM, DSG, BS2G, DSSO, DSBU, CBDPS, PIR). [@doi:10.1021/jasms.9b00085]
+3. Determine the cross-linking efficiency through binding energy and calculations.
+4. Determine the number is cross-links which then will help you know the molecular weight.
+
+There are still some challenges in XL-MS such as:
+1. minimum amount of cross-linked peptides in complex mixtures [@doi:10.1021/acs.analchem.7b04431]
+2. diversity of cross linked products [@doi:10.1021/acs.analchem.7b04431]
+
+However, further advancements will continue to minimize the impact of these issues.
+
+2. Hydrogen deuterium exchange mass spectrometry (HDX-MS)
+HDX-MS works by detecting changes in the mass of the ion in which the amide hydrogens of the protein backbone is exchanged with its surrounding solvents [@doi:10.1038/s41592-019-0459-y]. 
+The rate of exchange depends on the protein solvent accessible surface area, dynamics, and the properties of the amino acid sequence [@doi:10.1038/s41592-019-0459-y; @doi:10.1039/C0CS00113A; @doi:10.1016/S0065-3233(08)60129-1; @doi:10.1017/S0033583500005217].
+Although using D2O as a solvent to make deuterium labeled sample is simple, HDX-MS requires a number of controls to ensure that experimental conditions are done in an unbiased manner [@doi:10.1038/s41592-019-0459-y; @doi:10.1586/epr.10.109; @doi:/10.1007/s13361-012-0377-z; @doi:10.4155/bio.15.46]. 
+If done correctly, HDX can produce precise protein structure measurements with high reproducibility.
+Masson **et al.** gave recommendations on how to prep samples, conduct data analysis, and present findings in a detailed stepwise manner that will be useful to have if you are new to using this technique [@doi:10.1038/s41592-019-0459-y].
+
+3. Fast photochemical oxidation of proteins (FPOP) [@doi:10.1074/jbc.REV119.006218].
+FPOP is a laser based hydroxyl radical protein footprinting MS method that relies on the irreversible labeling of solvent-exposed amino acid side chains by hydroxyl radicals in order to understand structure elucidation of proteins.
+A laser produces 248 nm light to cause hydrogen peroxide to become hydroxyl radicals [@doi:10.1016/j.jasms.2005.09.008; @doi:10.1016/j.ijms.2006.08.018]. 
+The flow rate of solution through the capillary and laser frequency are adjusted so that each protein molecule is radiated only once. 
+After they are radiated, the sample is collected to a tube that has catalase and free methionine in the buffer so that the H2O2 and hydroxyl radical are quenched to prevent further unwarranted oxidation, which may then modify residues that become exposed due unfolding after the initial labeling. 
```

Comment:
```suggestion
After they are irradiated, the sample is collected to a tube that has catalase and free methionine in the buffer so that the H2O2 and hydroxyl radical are quenched to prevent further unwarranted oxidation, which may then modify residues that become exposed due unfolding after the initial labeling. 
```